schemas/contacts-schemas.js

```diff
@@ -0,0 +1,12 @@
+const Joi = require("joi");
+
+const contactsAddSchema = Joi.object({
+  name: Joi.string().required(),
```

Comment:
schema should be valid. Now name, email and phone might be just any string for example "sdasda" isn't valid email, right? Find a way to handle this